phpBB/phpbb/console/command/db/console_migrator_output_handler.php

```diff
@@ -54,11 +54,11 @@ public function write($message, $verbosity)
 		{
 			$translated_message = call_user_func_array(array($this->user, 'lang'), $message);
 
-			if ($verbosity === migrator_output_handler_interface::VERBOSITY_NORMAL)
+			if ($verbosity === OutputInterface::VERBOSITY_NORMAL)
```

Comment:
Because the output handler for the CLI worked with symfony constants, the migrator however did with ours.
And since they would otherwise be the same I changed all of them to use Symfony constants.
Furthermore I changed ours to be like the Symfony constants for BC reasons. I suggest to remove them in the future, though.